iconservice/icx/issue/issue_engine.py

```diff
@@ -91,8 +102,25 @@ def issue(self,
 
             total_issue_amount += issue_data_in_db[group_key]["value"]
 
-        # todo : issue amount = total_issue_amount - prev total transaction fee
-        # to_address: Address to be deposited
-        self._issue(context, to_address, total_issue_amount)
-        total_issue_event_log: 'EventLog' = self._create_total_issue_amount_event_log(total_issue_amount)
+        # todo : iiss_engine 에 calc 주기를 넘겨주는 method를 생성하거나 variable property를 생성
```

Comment:
comments in english

Reply:
I have removed this comment as having been solved